Source/Core/DolphinQt/Utils/Resources.cpp

```diff
@@ -70,10 +70,8 @@ void Resources::UpdatePixmaps()
 	m_pixmaps[TOOLBAR_FULLSCREEN].load(GIFN("fullscreen"));
 	m_pixmaps[TOOLBAR_SCREENSHOT].load(GIFN("screenshot"));
 	m_pixmaps[TOOLBAR_CONFIGURE].load(GIFN("config"));
-	m_pixmaps[TOOLBAR_PLUGIN_GFX].load(GIFN("graphics"));
-	m_pixmaps[TOOLBAR_PLUGIN_DSP].load(GIFN("dsp"));
-	m_pixmaps[TOOLBAR_PLUGIN_GCPAD].load(GIFN("gcpad"));
```

Comment:
No, they don't exist anymore. In DolphinWX, the GC and Wii controller configs were unified into a single Controllers config, and the Audio button was removed entirely since it's accessible from the main config.